docs/D-kit/Lidar_Based_Auto_Driving/virtual_lane_generation_cn.md

```diff
@@ -34,7 +34,16 @@
 |3|/apollo/sensor/gnss/ins_stat|
 |4|/apollo/sensor/lidar16/compensator/PointCloud2|
 
-- 为获取上述`channel`，需要启动`GPS`、`Localization`、`Velodyne`三个模块，`GPS`、`Localization`模块的启动请参考定位模块配置文档，`Velodyne`模块的启动请参考感知传感器集成文档。由于GNSS设备的限制，`/apollo/sensor/gnss/odometry`、`/apollo/sensor/gnss/ins_stat`这两个`channel`不能由GNSS设备直接给出，需要借助`/apollo/modules/tools/sensor_calibration/`下的两个脚本工具。在`localization`模块正常启动且输出`/apollo/localization/pose`数据时，在不同终端分别执行`python modules/tools/sensor_calibration/ins_stat_publisher.py`、`python modules/tools/sensor_calibration/odom_publisher.py`两个命令，便可以分别产生`/apollo/sensor/gnss/ins_stat`、`/apollo/sensor/gnss/odometry`这两个`channel`。
+- 为获取上述`channel`，需要启动`GPS`、`Localization`、`Velodyne`三个模块，`GPS`、`Localization`模块的启动请参考定位模块配置文档，`Velodyne`模块的启动请参考感知传感器集成文档。由于GNSS设备的限制，`/apollo/sensor/gnss/odometry`、`/apollo/sensor/gnss/ins_stat`这两个`channel`不能由GNSS设备直接给出，需要借助`/apollo/modules/tools/sensor_calibration/`下的两个脚本工具。在`localization`模块正常启动且输出`/apollo/localization/pose`数据时，另外开启两个不同终端进入docker后在/apollo根目录下分别执行
+```bash
+  bazel build modules/tools/sensor_calibration:ins_stat_publisher
+  bazel run modules/tools/sensor_calibration:ins_stat_publisher
+```
+```bash
+  bazel build modules/tools/sensor_calibration:odom_publisher
+  bazel run modules/tools/sensor_calibration:odom_publisher
```

Comment:
Using `./apollo.sh build tools` to build
and `./bazel-bin/module/tools/xxx/odom_publisher` to run.